app/views/includes/spinner.njk

```diff
@@ -0,0 +1,7 @@
+{% macro spinner(params) %}
+  <div class="spinner"></div>
+  {% if params.text %}
+    <h3 class="govuk-heading-m" style="text-align: center">{{ params.text }}</h3>
```

Comment:
remove this in-line style (I know I added this but that was back when this was a WIP)

Reply:
Fixed